.github/workflows/testing.yml

```
@@ -0,0 +1,47 @@
+        uses: otymko/setup-onescript@v1.5
+        with:
+          version: ${{ steps.extract_oscript_version.outputs.version }}
+
+      - name: Установка зависимостей
+        run: |
+          opm install opm
+          opm install -l --dev
+
+      - name: Тестирование
+        run: |
+          oscript src\cli\main.os e
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)